### ISSUE TITLE
Sorting between section

### ISSUE THREAD

**Issue by jfsigrist** [davraamides/todotxt-mode#12]
Hi, I'm not quite sure about the section usage (--) but while I separated the todo file in section the sorts commands mixes everything up.

Should the sort be limited to the current section, or maybe sort but only within each section?

**Comment by davraamides** [comment]
The section divider is meant as a way to separate the file into two areas. The top area is for todo tasks as normal and everything below the separator (by default, a line with just -- on it) is ignored. So sorting and decorating should just occur on the lines above the separator. Like this:

A task
Another task
--
Any text you want and it is ignored by the remaining commands.

Is this not happening? If you give me an example, I will look into it. I added this behavior simply because I like to collect notes in the same file. But you don't need to use the feature if it's not useful for you.

-David

> On Jan 16, 2020, at 9:52 PM, jfsigrist <notifications@github.com> wrote:
> 

﻿
Hi, I'm not quite sure about the section usage (--) but while I separated the todo file in section the sorts commands mixes everything up.

Should the sort be limited to the current section, or maybe sort but only within each section?
